### PR TITLE
Update Installation and Usage instructions

### PR DESCRIPTION
**Pull request by dylan-hoefsloot**
This changes were required to get the package working for me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,7 +70,7 @@ In your project locate this file `{{your_project}}_web.ex`, and add:
   defp html_helpers do
     quote do
       ...
-      use Phoenix.LiveView.Components.MultiSelect   ## <--- add this line
```
Comment (saleyn) [inline README.md]: Why this change?  The implementation of `use` calls the import with only one `[multi_select: 1]` function.

**Review by saleyn — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dylan-hoefsloot) [reply README.md]: It didn't work for me with `use` 🤷
